compliance/github/repository_naming/README.md

```diff
@@ -0,0 +1,32 @@
+## GitHub.com Repository Naming
```

Comment:
As tthe other ones, i would name this something more specific about what it checks for "Unpermitted Repo... name" or whatever

Reply:
Renamed to `GitHub.com Unpermitted Repository Names` and modified description in commit b7dba63